### PR TITLE
Fill out "What's in the template?" in the README

### PR DESCRIPTION
**Pull request by sharplet**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,7 +9,16 @@ Inspired by [suspenders], and [django-template].
 
 ## What's in the template?
 
-- TODO
+- [yarn][]
+- [redux][]
```
Comment (jakecraige) [inline README.md]: What about adding in `redux-thunk` as well?

File: README.md [hunk 2]
```diff
@@ -9,7 +9,16 @@ Inspired by [suspenders], and [django-template].
...
+- [react-navigation][]
+- [jest][]
+- ...and an example app to use as a starting point
```
Comment (jakecraige) [inline README.md]: Eslint and remove-redux-devtools?

**Comment by sharplet** [comment]
Done 👍

File: README.md [hunk 3]
```diff
@@ -9,7 +9,21 @@ Inspired by [suspenders], and [django-template].
 
 ## What's in the template?
 
-- TODO
+- [yarn][]
+- [redux][] and [react-redux][]
```
Comment (jakecraige) [inline README.md]: Is `redux-thunk` what you meant? 😄

Comment (sharplet) [inline README.md]: smh

Comment (sharplet) [inline README.md]: Fixed 👍

**Pull request merged**